beep/structure/biologic.py

```diff
@@ -119,14 +119,14 @@ def from_file(cls, path, mapping_file=None):
         data["data_point"] = list(range(1, len(raw["cycle number"]) + 1))
 
         data = pd.DataFrame(data)
-        data.loc[data.step_index % 2 == 0, "charge_capacity"] = abs(
-            data.charge_capacity
-        )
-        data.loc[data.step_index % 2 == 1, "charge_capacity"] = 0
-        data.loc[data.step_index % 2 == 1, "discharge_capacity"] = abs(
-            data.discharge_capacity
-        )
-        data.loc[data.step_index % 2 == 0, "discharge_capacity"] = 0
+        # data.loc[data.step_index % 2 == 0, "charge_capacity"] = abs(
+        #     data.charge_capacity
+        # )
+        # data.loc[data.step_index % 2 == 1, "charge_capacity"] = 0
+        # data.loc[data.step_index % 2 == 1, "discharge_capacity"] = abs(
+        #     data.discharge_capacity
+        # )
+        # data.loc[data.step_index % 2 == 0, "discharge_capacity"] = 0
```

Comment:
@shijingsun-TRI Is this code no longer needed even for the existing csv files?

Reply:
That's not needed for biologic csv files we are working with now.

Reply:
Would it still support the existing biologic files or does that code truly have no use?

Reply:
I think the assumption that the charge and discharge steps will alternate does not generalize since there might be rest steps, etc. Also it looks like all of the files we get so far are resetting the charge and discharge capacity values, so this does not seem to be necessary any more.

Reply:
I don't see the use of these lines even for the existing biologic files.

Reply:
Ok in that case I'd vote to just remove them rather than have them as comments.